text_extensions_for_pandas/io/watson/test_nlu.py

```diff
@@ -247,10 +247,10 @@ def test_entity_mentions(self):
         self.assertEqual(
             repr(result["entity_mentions"]),
             textwrap.dedent("""\
-                       type           text                         span  confidence
-                0    Person  Steven Wright    [64, 77): 'Steven Wright'    0.998251
-                1    Person  Steven Wright  [177, 190): 'Steven Wright'    0.998251
-                2  Location         Alaska         [138, 144): 'Alaska'    0.999498""")
+                       type                         span  confidence
```

Comment:
I removed the text column because it's redundant with the span, similar to other DataFrames with spans

Reply:
In this case, the `text` column actually serves a purpose. That column is part of the join key for joining entity mentions in the entity mentions DataFrame with the document-level information (sentiment, emotion, and document-level confidence) in the entities DataFrame. The whitespace and capitalization of the value in the `text` column could potentially be different from the covered text of the span.

The current implementation in NLU tends to treat multiple mentions of the same entity with slightly different surface forms as completely different entities, but I don't think we can assume that will always be the case. NLU ought to treat `John Smith`, `John  Smith` and `john Smith` as the same entity for the purposes of computing things like sentiment and emotion.

How about if we keep this column in the entity mentions DataFrame, but rename the column from "text" to something like "name" across both entity-related DataFrames?

Reply:
Ah ok, that makes sense to keep the `text` column then. I think we should keep it as `text` though and not rename to be consistent with the Watson response, that ok with you?

Reply:
Works for me. Thanks!